solr/test-framework/src/java/org/apache/solr/cloud/MiniSolrCloudCluster.java

```diff
@@ -299,8 +299,9 @@ public MiniSolrCloudCluster(
             .withUrl(zkServer.getZkHost())
             .withTimeout(AbstractZkTestCase.TIMEOUT, TimeUnit.MILLISECONDS)
             .build()) {
-      if (!zkClient.exists("/solr/solr.xml", true)) {
-        zkClient.makePath("/solr/solr.xml", solrXml.getBytes(Charset.defaultCharset()), true);
+      // NOCOMMIT: Create chroot here, not sure why it is now necessary?
+      zkClient.makePath("/solr", false, true);
```

Comment:
Hmm, you would hope the code in [SOLR-7642](https://issues.apache.org/jira/browse/SOLR-7642) would make this unnecessary... Or is it because of the `zkClient.MakePath()` code below? weird since that is the same code as was deleted for `solr.xml`...

Reply:
Earlier line 303 always created `/solr/solr.xml`, so when that is removed, and security is not enabled, we need to create `/solr` manually.